core/resources/intent_example.xml

```diff
@@ -0,0 +1,83 @@
+<h:html xmlns:h="http://www.w3.org/1999/xhtml" xmlns:orx="http://openrosa.org/jr/xforms" xmlns="http://www.w3.org/2002/xforms" xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns:jr="http://openrosa.org/javarosa" xmlns:vellum="http://commcarehq.org/xforms/vellum">
```

Comment:
I added this file so that one can easily (manually) test/see what the PR is achieving (see example usage in PR description). I should probably just write real tests, but when I started this PR the unit testing framework wasn't up and running again.